### PR TITLE
Feat: Webapp Sparrow Edge - Teams & Workspaces

### PR DESCRIPTION
**Pull request by aakashreddy-p**
## Description

Please do not leave this blank 
I have made some changes to provide support for the web app Sparrow Edge and made UI changes as well[#2290].
## What type of PR is this? (check all applicable)
- [ ] 🍕 Feature
## Have you tested locally?
- [ ] 👍 yes

### CODE REVIEW

File: packages/@sparrow-teams/src/features/team-explorer/layout/TeamExplorer.svelte [hunk 1]
```diff
@@ -234,7 +234,11 @@
             <span
               class="ms-3 my-auto ellipsis overflow-hidden heading"
               style="font-size: 28px;"
-              >{openTeam?.name || ""}
+              >{#if isGuestUser}
```
Comment (Astitva877) [inline packages/@sparrow-teams/src/features/team-explorer/layout/TeamExplorer.svelte:237]: We don't need this if else condition, when we create dummy team for guest user we already that with "Team" name so no  need for this condition.

Comment (aakashreddy-p) [reply packages/@sparrow-teams/src/features/team-explorer/layout/TeamExplorer.svelte]: I will fix it.

File: packages/@sparrow-common/src/components/header/sub-component/PopupHint.svelte [hunk 2]
```diff
@@ -74,4 +77,20 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    height: 25px;
+    width: 335px;
+    font-size: 12px;
+    background-color: var(--bg-tertiary-400);
+    left: -20px;
+  }
```
Comment (itsmdasifraza) [inline packages/@sparrow-common/src/components/header/sub-component/PopupHint.svelte:95]: you can handle this dynamically, no need to add check for desktop and web, currently it is hardcoded.

File: packages/@sparrow-teams/src/features/team-explorer/components/team-navigator/TeamNavigator.svelte [hunk 3]
```diff
@@ -29,7 +30,7 @@
               : ''}"
             style="font-size: 12px; font-weight:700"
             ><span>{tab.name}</span>
-            {#if tab.count}
+            {#if tab.count && !isGuestUser}
```
Comment (itsmdasifraza) [inline packages/@sparrow-teams/src/features/team-explorer/components/team-navigator/TeamNavigator.svelte:33]: this is not required

Comment (aakashreddy-p) [reply packages/@sparrow-teams/src/features/team-explorer/components/team-navigator/TeamNavigator.svelte]: okay I will fix it

File: packages/@sparrow-common/src/components/header/sub-component/PopupHint.svelte (hunk 2 above)
Comment (aakashreddy-p) [reply packages/@sparrow-common/src/components/header/sub-component/PopupHint.svelte]: I will fix it

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**